src/Symfony/Bundle/SecurityBundle/DependencyInjection/SecurityExtension.php

```diff
@@ -161,6 +161,18 @@ public function load(array $configs, ContainerBuilder $container)
 
         $container->registerForAutoconfiguration(VoterInterface::class)
             ->addTag('security.voter');
+
+        $this->addClassesToPreload([
+            'Symfony\Component\HttpFoundation\RequestMatcher',
+            'Symfony\Component\Security\Core\Authorization\ExpressionLanguageProvider',
```

Comment:
what if ExpressionLanguage is not installed ? wouldn't it break it ?

Reply:
nope: it would just skip it